common/filestore_utils.py

```diff
@@ -35,11 +35,12 @@ def _using_gsutil():
     from common import gsutil as filestore_utils_impl
 
 
-def cp(source, destination, recursive=False, parallel=False):  # pylint: disable=invalid-name
+def cp(source, destination, recursive=False, must_exist=True, parallel=False):  # pylint: disable=invalid-name
```

Comment:
I guess this and related changes can be separate into another PR and makes this PR easier to focus on queue implementation bug fixing.

Reply:
I think it makes sense to keep here.

Reply:
I'm actually merging with master, I think this change will go away when I'm done.